### DISCUSSION TITLE
DateTime and Date fields

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#17190] [How to]
Hello, I have a question, why is the day and time field different from the standard date field on the dashboard?
![Untitled](https://github.com/YetiForceCompany/YetiForceCRM/assets/75428743/1c2950ab-c635-4b02-a55e-2e8efa22f432)

yetiforce version :  6.1.0

**Comment by Akaki-1991** [comment]
.

**Comment by Jianbin719** [comment]
look into the table in "tiger_field" and try to compare the UItype and typeofdata??

**Reply by Akaki-1991** [reply]
UItype and  typeofdata is different, because it is different field types.
